app/assets/javascripts/discourse/app/components/user-nav/dropdown-list.hbs

```diff
@@ -0,0 +1,18 @@
+<li class={{concat @class " user-nav-dropdown-list-item"}}>
```

Comment:
We have a `concat-class` helper here that Joffrey added https://github.com/discourse/discourse/commit/0760b249ffe39a9ff57b7e36b6304e63a5082d7e , I think the main benefit is that it handles null/undefined better?

Reply:
O nice :+1: